modules/monitoring-attaching-additional-labels-to-your-time-series-and-alerts.adoc

```diff
@@ -6,7 +6,10 @@
 [id="attaching-additional-labels-to-your-time-series-and-alerts_{context}"]
 = Attaching additional labels to your time series and alerts
 
-Using the external labels feature of Prometheus, you can attach custom labels to all time series and alerts leaving Prometheus.
+You can use the external labels feature of Prometheus to attach custom labels to all time series and alerts leaving the Prometheus instance used for core {product-title} monitoring.
+
+If you enable monitoring for user-defined projects, Prometheus manages metrics and Thanos Ruler manages alerting and recording rules.
+Therefore, you can only attach external labels for metrics and not for alerting and recording rules when you configure the `prometheus` component in the `user-workload-monitoring-config` `ConfigMap` object.
```

Comment:
the description for UWM seems wrong, tested with 4.10.52 , created user project, configuration see from: http://pastebin.test.redhat.com/1091474
enabled UWM and configure externalLabels for UWM prometheus, alerts from thanos-ruler don't have externalLabels, but the recording/alerting rules in thanos-ruler have externalLabels, see from: http://pastebin.test.redhat.com/1091477
same result for 4.13